metabase/metabase
# Bottom corner radius on buttons looks weird on new dashboard grid

Closes https://github.com/metabase/metabase/issues/31587

### Description

Most likely because of the new dashboard grid, action buttons were having their buttons cut off when a user would be editing or viewing a dashboard with a sidebar open. This bug was only happening on a certain breakpoint so I removed the padding from that button for that breakpoint.

I also added tests at three different breakpoints to make sure that action buttons don't overflow again.

### How to verify

_Current behavior_
When editing a dashboard, click "Add action." On `master`, the button will be cut off at the bottom. 
When viewing a dashboard with the info sidebar open, action buttons will be cut off at the bottom.

With this PR, these behaviors should disappear.

### Checklist
